src/components/Issues/Issues.css

```diff
@@ -1 +1,15 @@
+.card-width {
```

Comment:
whoa dude! Slow down. This has potential to break everybody else's widgets! these card styles are semantic stuff, right?